Makefile

```diff
@@ -601,6 +595,44 @@ HIGHEND_SRC = \
             telemetry/mavlink.c \
             telemetry/esc_telemetry.c \
 
+NON_RT_SRC = \
+            drivers/serial_escserial.c \
+            io/serial_cli.c \
+            io/serial_4way.c \
+            io/serial_4way_avrootloader.c \
+            io/serial_4way_stk500v2.c \
+            msp/msp_serial.c \
+
+LOW_PRIO_SRC = \
```

Comment:
`LOW_PRIO_SRC` is a misnomer. To the casual reader it looks like it means these source files are less important than the other ones.

I suggest `SPEED_OPTIMISED_SRC` and `SIZE_OPTIMISED_SRC`.